src/frontends/jax/src/op/select_n.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/jax/node_context.hpp";
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/gather_elements.hpp"
+
+#include "utils.hpp";
+
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace jax {
+namespace op {
+
+OutputVector translate_select_n(const NodeContext& context) {
+    num_inputs_check(context, 2);
+    auto num_inputs = static_cast<int>(context.get_input_size());
+    Output<Node> which = context.get_input(0);
+    if (which.get_element_type() == element::boolean) {
+        which = std::make_shared<v0::Convert>(which, element::i32);  
+    }
+    OutputVector cases_vector(num_inputs - 1);
+    for(int ind = 1; ind < num_inputs; ++ind) {
+        cases_vector[ind - 1] = context.get_input(ind);
+    }
+    
+    Output<Node> cases = std::make_shared<v0::Concat>(cases_vector, 0);
+    auto which_shape = which.get_shape();
```

Comment:
your previous impls with ShapeOf was correct. Just don't use broadcast operation.
Now you face limitation with `get_shape` that raises exception for dynamic shape.

Reply:
okay done